docs/sql-migration-guide.md

```diff
@@ -225,6 +225,8 @@ license: |
   - Since Spark 3.0, when casting string value to integral types, including tinyint, smallint, int and bigint type, the leading and trailing white spaces(<= ACSII 32) will be trimmed before convert to integral values, e.g. `cast(' 1 ' as int)` results `1`. In Spark version 2.4 and earlier, the result will be `null`.
 
   - Since Spark 3.0, when casting string value to date, timestamp and interval values, the leading and trailing white spaces(<= ACSII 32) will be trimmed before casing, e.g. `cast('2019-10-10\t as date)` results the date value `2019-10-10`. In Spark version 2.4 and earlier, only the trailing space will be removed, thus, the result is `null`.
+  
+  - Since Spark 3.0, numbers written in scientific notation(e.g. `1E2`) would be parsed as Double. In Spark version 2.4 and earlier, they're parsed as Decimal. To restore the behavior before Spark 3.0, you can set `spark.sql.legacy.exponentLiteralAsDecimal.enabled` to `true`.
```

Comment:
nit: `numbers` -> `numeric literals`?

Reply:
Wait, wait. `numeric` represents **exact** numeric/number in pg sql. But we may wish an approximate numeric/number here as we want to parse it as Double. So, in order to not make user be confused with these concepts, I'd prefer to original way. WDYT ? @maropu

Reply:
AFAIK `numeric` is a data type defined by SQL that has precision and scale. This type is also implemented in SQL server and pgsql.

I think `number` is less confusing here.

Reply:
Ur, I was mistaken. I meant number literals. Anyway, it doesn't matter. Thanks, guys.